### PR TITLE
feat(icons): expose processIcons utility

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by waterplea** [comment]
`process-icons.js` is used on demo:
https://taiga-ui.dev/icon-set
So we shouldn't get rid of it completely. Or come up with some solution to let people run it on their icon sets and document it.

**Review by waterplea — APPROVED** [review]

**Comment by splincode** [comment]
@waterplea @nsbarsukov @vladimirpotekhin please, review me)

File: projects/demo/src/modules/customization/icon-set/examples/1/process-icons.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+```js
+// touch process-icons.js # paste code which below to this file
```
Comment (waterplea) [inline projects/demo/src/modules/customization/icon-set/examples/1/process-icons.md:2]: I don't get this line )

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/customization/icon-set/examples/1/process-icons.md (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/customization/icon-set/examples/1/process-icons.md]: sorry, fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**